check_has_owner.go

```diff
@@ -1,28 +1,28 @@
 package opslevel
 
 type ServiceOwnershipCheckFragment struct {
-	RequireContactMethod *bool                             `graphql:"requireContactMethod"`
-	ContactMethod        *ServiceOwnershipCheckContactType `graphql:"contactMethod"`
```

Comment:
enum.go doesn't generate this enum and it seems like contact type is the same thing so consolidating this.